src/edge/simulator.ts

```diff
@@ -106,9 +104,6 @@ export class Simulator {
                     type = "upgradeStandalone";
                 }
             }
-        } catch (error) {
-            message =  Constants.needSimulatorInstalledMsg;
-            type = "install";
         }
 
         if (message) {
```

Comment:
Do we need to check this now? It seems if come here there should be message, else it already return?

Reply:
Yes, we need to check it, because if user has already installed the latest version, we do not need to take any other actions.

Reply:
another way is using return: 
![image](https://user-images.githubusercontent.com/5545529/68281304-5a74bc80-00b2-11ea-82be-6c86c909d7b7.png)

Reply:
Seems already return, so no need to check?